app/scripts/controllers/events_controller.js

```diff
@@ -0,0 +1,40 @@
+(function(){
+  'use strict';
+
+  angular
+    .module('app.controllers')
+    .controller('EventsController', EventsController);
+
+  EventsController.$inject = ['DataFactory', '$firebaseArray'];
+
+  function EventsController(DataFactory, $firebaseArray){
+    var vm = this;
+    var eventRef = DataFactory("events");
+    var currentDate = new Date();
+    var startDate = currentDate.setHours(0,0,0,0);
+    var endDate = currentDate.setHours(23,59,59,999);
+    vm.timeFrom =  new Date(startDate);
+    vm.timeTo =  new Date(endDate);
+
+    vm.events = $firebaseArray(eventRef.orderByChild("timeTo").startAt(startDate).endAt(endDate));
+    vm.events.$loaded().then(function() {
+      vm.rooms = vm.events.map(function(k){return k.roomName});
+    });
+
+    vm.change = function(){
+      var startDate = Date.parse(new Date(vm.timeFrom));
```

Comment:
Because timeFrom, timeTo is stored integer format in databases -> we need convert string to integer